examples/Payment/Create.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Examples\Payment;
+
+// Step 1: Require the Composer library
+require_once '../../vendor/autoload.php';
+
+use MercadoPago\Client\Common\RequestOptions;
+use MercadoPago\Client\Payment\PaymentClient;
+use MercadoPago\Exceptions\MPApiException;
+use MercadoPago\MercadoPagoConfig;
+
+// Step 2: Set the production or sandbox access token
+MercadoPagoConfig::setAccessToken('<ACCESS_TOKEN>');
+// Step 2.1 (optional - default is SERVER): Set your runtime enviroment from MercadoPagoConfig::RUNTIME_ENVIROMENTS
+// In case you want to test in your local machine first, set runtime enviroment to LOCAL
+MercadoPagoConfig::setRuntimeEnviroment(MercadoPagoConfig::LOCAL);
+
+// Step 3: Initialize the API client
+$client = new PaymentClient();
+
+try {
+    // Step 4: Create the request array
+    $request = [
+        "transaction_amount" => 1000,
+        "description"        => "Título do produto",
+        "payment_method_id"  => "master",
+        "token"              => "{{CARD_TOKEN}}",
+        "installments"       => 1,
+        "application_fee"    => null,
+        "binary_mode"        => true,
+        "capture"            => true,
+        "external_reference" => "Pedido01",
+        "metadata"           => new \stdClass(),
+        "notification_url"   => "http://requestbin.fullcontact.com/1ogudgk1",
```

Comment:
[pp] Não seria melhor deixar a URL do Mercado Pago?
```suggestion
        "notification_url"   => "https://www.mercadopago.com",
```